src/services/pairing/utils.ts

```diff
@@ -28,3 +32,28 @@ export const killPairingSession = (connector: InstanceType<typeof WalletConnect>
 export const isPairingSupported = (disabledWallets?: string[]) => {
   return !!disabledWallets?.length && !disabledWallets.includes(CGW_NAMES[WALLET_KEYS.PAIRING] as string)
 }
+
+export const _hasPairingSessionExpired = (session: IWalletConnectSession): boolean => {
```

Comment:
nit: I would name this `_hasExpiredPairingSession` or `_isPairingSessionExpired`